### ISSUE TITLE
Terminal Bugs (Minor) - Past

### ISSUE THREAD

**Issue by scottishbay** [cockpit-project/cockpit#20532]
Need to past twice to past and what I past is cut short.
After a right click on terminal I see copy/past. When I select past we get another prompt to past. Selecting the second prompt works.
But what I past is cut short. 44 of 66 lines of bash code to be exact.

**Comment by jelly** [comment]
You reported this originally on our website bug tracker, but I assume this is about a cockpit terminal page bug? What version of Cockpit and what is exactly the issue?

IIF you copy a large text, is not everything pasted?

**Comment by scottishbay** [comment]
Yes, it's cockpit terminal page. The version is 316-1~bpo23.10.1. 44 lines of anything is all it pasts. Not everything is pasted no.
